src/credential/sd_jwt_vc.rs

```diff
@@ -0,0 +1,169 @@
+use base64::prelude::*;
+use ssi::{
+    claims::{
+        jwt::{AnyClaims, JWTClaims},
+        sd_jwt::{RevealedSdJwt, SdJwtBuf},
+        vc::{v1::Credential as _, v2::Credential as _},
+        JwsString,
+    },
+    prelude::AnyJsonCredential,
+};
+use uuid::Uuid;
+
+use crate::{CredentialType, KeyAlias};
+
+use super::VcdmVersion;
+
+#[derive(uniffi::Object, Debug, Clone)]
+/// A verifiable credential secured as a JWT.
+pub struct SdJwtVc {
+    id: Uuid,
+    sd_jwt: JwsString,
+    credential: AnyJsonCredential,
```

Comment:
This can only be a `ssi::claims::vc::v2::JsonCredential`.

Reply:
This has been removed, but possibly may be re-added to support presentation in the near future.